bot/resources/tags/comparison.md

```diff
@@ -0,0 +1,11 @@
+#### Assignment vs. Comparison
+
+The equals operator (=) is used to assign variables.
+```python
+x = 5
+```
+The double equals operator (==) is used to compare values.
```

Comment:
```suggestion
The double equals operator (`==`) is used to compare values.
```
What do you think about adding code blocks around the `==`, since it's referring to code?

Reply:
Yep